src/plugins/plugin.autoplay.js

```diff
@@ -103,9 +103,9 @@ BookReader.prototype.autoToggle = function(overrides) {
       if (this.animating) return;
 
       if (Math.max(this.twoPage.currentIndexL, this.twoPage.currentIndexR) >= this.lastDisplayableIndex()) {
-        this.flipBackToIndex(1); // $$$ really what we want?
+        this.prev({ triggerStop: false }); // $$$ really what we want?
```

Comment:
what's `triggerStop`?

Reply:
Darn I was trying to sneak this in :P Probably something we should clean up later, cause it's a little weird. I added a new param whether it should fire the "Stop" event, because previous, flipBackToIndex wouldn't fire, but I deleted that public method. So I added a flag here.